embedchain/config/apps/base_app_config.py

```diff
@@ -1,7 +1,7 @@
 import logging
 
-from embedchain.config.BaseConfig import BaseConfig
-from embedchain.config.vectordbs import ElasticsearchDBConfig
+from embedchain.config.base_config import BaseConfig
+from embedchain.config.vectordbs import elasticsearch_db_config
```

Comment:
maybe `vector_db` if we want to nitpick

Reply:
cause we also use `elasticsearch_db`

Reply:
I also changed the folder names `vectordb` to `vector_db` and `vectordbs` to `vector_dbs`